src/locale/localed-util.c

```diff
@@ -56,13 +56,91 @@ static const char* systemd_language_fallback_map(void) {
         return SYSTEMD_LANGUAGE_FALLBACK_MAP;
 }
 
+static void x11_context_clear(X11Context *xc) {
+        assert(xc);
+
+        xc->layout  = mfree(xc->layout );
+        xc->options = mfree(xc->options);
+        xc->model   = mfree(xc->model  );
+        xc->variant = mfree(xc->variant);
```

Comment:
aligning the `=` makes sense to me, but aligning the `)` goes a little bit too far I think... dunno...